.github/contributing.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to nim.nvim
```

Comment:
```suggestion
### Contributing to nim.nvim
```

Increasing the level because it looks a bit nicer when rendered on github and that's also what used for the readme.